docs/samples/tasks.yaml

```diff
@@ -12,6 +12,10 @@
   # The DAP query type. See below for an example of a fixed-size task
   query_type: TimeInterval
 
+  # The differential privacy strategy. Note that it has to be supported
+  # by the selected VDAF.
+  dp_strategy: !NoDifferentialPrivacy {}
```

Comment:
These parts of the file are ignored by `VdafInstance` deserialization. Note that for a task YAML file that uses Prio3FixedPointBoundedL2VecSum, the `dp_strategy` key would be nested inside the tagged map under the `vdaf` key, not at the top level of the task. We should roll back the changes to this file to avoid confusion.

When we eventually set up optional DP strategies for standardized VDAFs, we'll want to configure deserialization with defaults (of NoDifferentialPrivacy) for backwards compatibility with existing task YAML files, database JSON columns, and aggregator API JSON requests/responses.

Reply:
We removed our changes.